tests/behat/features/bootstrap/AlbumContext.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Drupal\social\Behat;
+
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Behat\Gherkin\Node\TableNode;
+use Behat\Mink\Exception\ElementNotFoundException;
+use Behat\MinkExtension\Context\RawMinkContext;
+use Drupal\DrupalExtension\Context\DrupalContext;
+use Drupal\DrupalExtension\Context\MinkContext;
+
+/**
+ * Defines test steps around the usage of albums.
+ */
+class AlbumContext extends RawMinkContext {
+
+  use NodeTrait;
+  use GroupTrait;
+
+  private const CREATE_PAGE = "/node/add/album";
+
+  /**
+   * Keep track of what we created, so we can use them for validating.
+   */
+  private array $created = [];
+
+  /**
+   * Provide help filling in the WYSIWYG editor.
+   */
+  private CKEditorContext $cKEditorContext;
+
+  /**
+   * The Drupal mink context is useful for validation of content.
+   */
+  private MinkContext $minkContext;
+
+  /**
+   * The Drupal context which gives us access to user management.
+   */
+  private DrupalContext $drupalContext;
+
+  /**
+   * Make some contexts available here so we can delegate steps.
+   *
+   * @BeforeScenario
+   */
+  public function gatherContexts(BeforeScenarioScope $scope) {
+    $environment = $scope->getEnvironment();
+
+    $this->cKEditorContext = $environment->getContext(CKEditorContext::class);
```

Comment:
Do you still want to keep CKEditorContext, while this is not used at the moment?